### ISSUE TITLE
wan2.1 vae finetuning script

### ISSUE THREAD

**Issue by yfpeng1234** [modelscope/DiffSynth-Studio#650]
Hi,
I finetuned wan2.1-1.3b-t2v on my own dataset. I use vae to encode/decode the original video and compare the original video with reconstructed video. I found that psnr is around 26, which is not very satisfying. I guess I need to finetune vae on my own datasets for few steps. But I can't find the finetuning script for wan2.1 vae.

**Comment by Artiprocher** [comment]
@yfpeng1234 We currently have not developed the training functionality for VAE.

Additionally, we have also noticed the shortcomings of Wan 2.1 VAE, such as slight jittering occurring every four frames. In some of our small-scale experiments, we were unable to resolve this issue through training.

**Comment by yfpeng1234** [comment]
Thanks for your reply and pointing that out. Look forward to further improvement

**Comment by yfpeng1234** [comment]
Hi @Artiprocher , sorry to trouble you. I encountered a problem when finetuning vae on my own dataset. According to my understanding, VAE consists of encoder, conv1, conv2, decoder. I finetune conv2 and decoder and freeze encoder and conv1. But seems that the loss curve jump up and down:

<img width="427" height="278" alt="Image" src="https://github.com/user-attachments/assets/9d3e7275-8b02-48b9-83aa-a40f25710845" />

**Comment by yangqy1110** [comment]
> 你好[@Artiprocher](https://github.com/Artiprocher)抱歉打扰您。我在用自己的数据集微调 VAE 时遇到了问题。据我了解，VAE 由编码器、conv1、conv2 和解码器组成。我微调了 conv2 和解码器，并冻结了编码器和 conv1。但损失曲线似乎上下跳动：
> 
> <img alt="图像" width="427" height="278" src="https://private-user-images.githubusercontent.com/135833420/469090831-9d3e7275-8b02-48b9-83aa-a40f25710845.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzI2OTk3NDYsIm5iZiI6MTc3MjY5OTQ0NiwicGF0aCI6Ii8xMzU4MzM0MjAvNDY5MDkwODMxLTlkM2U3Mjc1LThiMDItNDhiOS04M2FhLWE0MGYyNTcxMDg0NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMzA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDMwNVQwODMwNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZjczMTE3YjNhMWUyNGUwOWYyNDM1NDVjODIwMmJiNDMxYWU5MWZkMmQxNGU0NzRhN2MyY2U4NDY4MjQwNGY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.8kky7_j9uJwMC-1yezO84RY7zrdABU8s1esfrmOWOOc">
I am Qinyu Yang, currently working on training a Wan VAE Decoder. 

I am writing to see if we could exchange some insights, as I have encountered some challenges during my training process that seem quite tricky:

Loss Oscillation: I am experiencing significant loss oscillation on the training set.

Learning Rate Sensitivity: I noticed that if the learning rate is relatively high, the loss on the validation set tends to rise. Conversely, with a lower learning rate, the loss decreases very slowly, and the actual reconstruction quality remains limited.

Since you have experience with this specific model, I would greatly appreciate it if we could discuss your observations or any strategies you found effective (such as learning rate scheduling or specific initialization techniques).

I am committed to pushing this project forward and would love to stay in touch to learn from each other. Looking forward to hearing from you!

**Comment by yfpeng1234** [comment]
Dear  @yangqy1110 , 
Thanks for reaching out! That project was done 8 months ago and I can not find the training log in wandb. I'm not an expert in training video-gen-model's vae but I remember that using bigger bz and more gradient accumulation step may help. If there is any question, feel free to contact me.
Best
